user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+.. hint::
+
+    This capability does not influence a user's ability to :ref:`edit their own 
+    password <user-guide-getting-started-profile-password>` from the *"My User"* page.
+
+
+.. _admin-capabilities-merge:
+
+Merge Entities
+--------------
+
+If the capability is set to *"System"*, the user can :ref:`merge <user-guide-ui-components-grids-delete-merge>` 
+several records of the same entity. By default the only entity, records of which can be merged is 
+:ref:accounts <user-guide-accounts>, but if other mergebale entities are added the setting will be applied. 
+
+By default, this capability should be set to *"System"* for  usually used by sales representatives, 
+marketing associates, and other employees working with accounts.
+
+
+.. _admin-capabilities-outlook:
+
+Outlook Integration
+-------------------
+
+This capability is meaningful for Enterprise users only. If the capability is set to *"System"*, the user can 
+:ref:`download <user-guide-getting-started-my-user-outlook>` the latest version of the OroCRM add-in for 
+Outlook. 
+
+If :ref:`OroCRM is synchronized with Outlook <user-guide-synch-outlook>`, you can use the add-in to synchronize 
+contacts, tasks, and calendar events between OroCRM and your Outlook account. You can also associate emails to accounts, 
+contacts, opportunities, and cases, as well as create leads, opportunities, and cases in OroCRM based on emails from 
+the Outlook sidebar.
+
+This is a general capability that can improve the user experience for all Enterprise users who are using both OroCRM and 
+Outlook. 
+
+
+|
+
+.. image:: ./img/intro/user_outlook.png
+
+|
+
+
+.. _admin-capabilities-address-dic:
+
+Read Address Dictionaries
+-------------------------
+
+If the capability is set to *"System"*, the user can access countries, regions, and address types via the API. 
+It has to be set to *"System"* in order to support Lead creation via Outlook. This capability should be activated to 
+system administrators or integrators who are authorized to access OroCRM via the API.
+
+
+.. _admin-capabilities-search:
+
+Search
+------
+
+If the capability is set to *"System"*, the user can use the :ref:`search <user-guide-getting-started-search>` 
+functionality to quickly find specific records.
+
+This is a general capability that can improve the overall experience of all users.
+
+The setting does not influence the user's ability to :ref:`search by tag <user-guide-getting-started-search-tag>`.
+
+
+.. _admin-capabilities-campaign-emails:
+
+Send Campaign Emails
+--------------------
+
+With OroCRM's :ref:`email campaigns <user-guide-email-campaigns>`, the user can send personalized template-based emails 
+to multiple users. This capability doesn't effect the users' ability to define and edit the campaign settings and create 
+templaets, but on if this capability is set to *"System"*, the user can launch the camplaign, i.e. start 
+:ref:`sending emails <user-guide-email-campaigns-send>` specified by the campaign.
```

Comment:
Edit: With OroCRM's :ref:`email campaigns <user-guide-email-campaigns>`, the user can send personalized template-based emails to multiple users. This capability doesn't affect the user's ability to define and edit the campaign settings and create templates, but this capability has to be set to *"System"*in order for a user to launch a campaign, i.e.. start :ref:`sending emails <user-guide-email-campaigns-send>` specified by the campaign.